src/compliance/ComplianceInterface.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.4.19;
 /// @title Compliance Interface Contract
 /// @author Melonport AG <team@melonport.com>
 /// @notice This is to be considered as an interface on how to access the underlying Compliance Contract
-interface ComplianceInterface {
```

Comment:
Just wondering if we can change back `contract` to `interface`, or if there is a reason why this change was made.
We are trying to use `interface` where possible now.